cluster-autoscaler/cloudprovider/gce/gce_cloud_provider.go

```diff
@@ -303,3 +307,27 @@ func (mig *gceMig) TemplateNodeInfo() (*schedulercache.NodeInfo, error) {
 	nodeInfo.SetNode(node)
 	return nodeInfo, nil
 }
+
+// BuildGCE builds GCE cloud provider, manager etc.
+func BuildGCE(opts config.AutoscalingOptions, do cloudprovider.NodeGroupDiscoveryOptions, rl *cloudprovider.ResourceLimiter) cloudprovider.CloudProvider {
```

Comment:
nit: Following go convention this method should be called BuildCloudProvider. And the the call would look gce.BuildCloudProvider.

With that said I am not fan of the convention :)